server/src/schema/migrations/1750323941566-UnsetPrewarmDimParameter.ts

```diff
@@ -1,15 +1,21 @@
 import { Kysely, sql } from 'kysely';
 
-export async function up(db: Kysely<any>): Promise<void> {
-  const { rows } = await sql<{ db: string }>`SELECT current_database() as db;`.execute(db);
-  const databaseName = rows[0].db;
-  await sql.raw(`ALTER DATABASE "${databaseName}" RESET vchordrq.prewarm_dim;`).execute(db);
-}
+export async function up(qb: Kysely<any>): Promise<void> {
+  type Conf = { db: string; guc: string[] };
+  const res = await sql<Conf>`select current_database() db, to_json(setconfig) guc from pg_db_role_setting`.execute(qb);
+  if (res.rows.length === 0) {
+    return;
+  }
 
-export async function down(db: Kysely<any>): Promise<void> {
-  const { rows } = await sql<{ db: string }>`SELECT current_database() as db;`.execute(db);
-  const databaseName = rows[0].db;
-  await sql
-    .raw(`ALTER DATABASE "${databaseName}" SET vchordrq.prewarm_dim = '512,640,768,1024,1152,1536';`)
-    .execute(db);
+  await sql`alter database immich reset all;`.execute(qb);
```

Comment:
```suggestion
  await sql`alter database "${db}" reset all;`.execute(qb);
```
see https://github.com/immich-app/immich/issues/19374